Added helm changes in deployment docs (#1908)
flyteorg/flyte: rsts/deployment/aws/manual.rst

Installing Flyte
-----------------

> #. Add the flyte chart repo to helm

Reviewer: ```suggestion
#. Add Flyte chart repo to Helm
```

>   .. code-block::

Reviewer 2: In the rendered view this like like a quoted string... I think it has an extra space in the beginning of the line... Maybe this line or maybe the next line is the problem

     helm repo add flyteorg https://flyteorg.github.io/flyte

> #. Download eks values for helm

Reviewer: ```suggestion
2. Download eks values for Helm
```

>    .. tabbed:: Flyte Native scheduler

Reviewer: ```suggestion
   .. tabbed:: Flyte Native Scheduler
```

>       * Download eks helm values, By default it enable the flyte native scheduler:

Reviewer: ```suggestion
      * Download eks Helm values (it enables Flyte native scheduler by default)
```

        .. code-block:: bash

           curl -sL https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-eks.yaml

   .. tabbed:: AWS scheduler

>       * Download eks helm values for aws scheduler

Reviewer: ```suggestion
      * Download eks helm values for AWS scheduler
```

        .. code-block:: bash

           curl -sL https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-eks.yaml
           curl -sL https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-eks-override.yaml

> #. Update values

Reviewer: ```suggestion
3. Update values in the YAML file
```

     - ARN of the self-signed (or official) certificate
     - ``arn:aws:acm:us-east-2:173113148371:certificate/763d12d5-490d-4e1e-a4cc-4b28d143c2b4``

> #. Install Flyte

Reviewer: ```suggestion
4. Install Flyte
```

   .. tabbed:: Flyte Native scheduler

>       * Install Flyte with flyte native scheduler:

Reviewer: ```suggestion
      * Install Flyte with Flyte native scheduler
```

        .. code-block:: bash

           helm install -n flyte -f values-eks.yaml --create-namespace flyte flyteorg/flyte-core

   .. tabbed:: AWS scheduler

>       * Install Flyte with flyte aws scheduler

Reviewer: ```suggestion
      * Install Flyte with Flyte AWS scheduler
```

        .. code-block:: bash

           helm install -n flyte -f values-eks.yaml -f values-eks-override --create-namespace flyte flyteorg/flyte-core

> #. Verify all the pods have come up correctly

Reviewer: ```suggestion
#. Verify if all the pods have come up correctly
```

     - ARN of the self-signed (or official) certificate
     - ``arn:aws:acm:us-east-2:173113148371:certificate/763d12d5-490d-4e1e-a4cc-4b28d143c2b4``

4. Install Flyte

.. tabbed:: Flyte Native scheduler

  * Install Flyte with Flyte native scheduler:

    .. code-block:: bash

       helm install -n flyte -f values-eks.yaml --create-namespace flyte flyteorg/flyte-core

.. tabbed:: AWS scheduler

  * Install Flyte with Flyte AWS Scheduler

    .. code-block:: bash

>        helm install -n flyte -f values-eks.yaml -f values-eks-override --create-namespace flyte flyteorg/flyte-core

Reviewer 2: ```suggestion
       helm install -n flyte -f values-eks.yaml -f values-eks-override.yaml --create-namespace flyte flyteorg/flyte-core
```

Upgrading Flyte
---------------

   .. tabbed:: Flyte Native scheduler

>       * Install Flyte with flyte native scheduler:

Reviewer: ```suggestion
      * Install Flyte with Flyte native scheduler
```

        .. code-block:: bash

           helm upgrade -n flyte -f values-eks.yaml --create-namespace flyte flyteorg/flyte-core

   .. tabbed:: AWS scheduler

>       * Install Flyte with flyte aws scheduler

Reviewer: ```suggestion
      * Install Flyte with Flyte AWS scheduler
```
